cluster/gke/config-default.sh

```diff
@@ -34,4 +34,5 @@ ELASTICSEARCH_LOGGING_REPLICAS=1
 #   none     - No cluster monitoring setup 
 #   influxdb - Heapster, InfluxDB, and Grafana 
 #   google   - Heapster, Google Cloud Monitoring, and Google Cloud Logging
-ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-none}"
+#   standalone     - Heapster only. Metrics available via Heapster REST API.
+ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-standalone}"
```

Comment:
We just need to scrub the options that won't do anything. It's been on my backburner for a little while, but hasn't been all that important. Don't worry about it in this pr.